src/sentry/incidents/endpoints/organization_alert_rule_available_action_index.py

```diff
@@ -7,22 +7,25 @@
 
 from sentry import features
 from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.constants import SentryAppStatus
 from sentry.incidents.endpoints.bases import OrganizationEndpoint
 from sentry.incidents.endpoints.serializers import action_target_type_to_string
 from sentry.incidents.logic import (
+    get_alertable_sentry_apps,
     get_available_action_integrations_for_org,
     get_pagerduty_services,
 )
 from sentry.incidents.models import AlertRuleTriggerAction
 
 
-def build_action_response(registered_type, integration=None, organization=None):
+def build_action_response(registered_type, integration=None, organization=None, sentry_app=None):
```

Comment:
It feels like this method has increased in complexity recently. It could probably do with some basic tests

Reply:
Added a new test: https://github.com/getsentry/sentry/pull/20263/files#diff-00273bf260ceb575bf94a3fd1125d599R70